packages/utils/src/ethers/helpers.ts

```diff
@@ -0,0 +1,22 @@
+import type { CaipNetwork } from '@web3modal/scaffold'
```

Comment:
Again, if it is only for ethers or only for wagmi, it should not be in utils, only shared bits should go in here.